### ISSUE TITLE
Glass Pane bug when using optifabric

### ISSUE THREAD

**Issue by Lava2Dad** [Chocohead/OptiFabric#1107]
Connected Texture not working on glass panes facing north and west. Connected textures works just fine when running optifine only.

**Comment by V4Qur** [comment]
![2023-07-16_22 44 49](https://github.com/Chocohead/OptiFabric/assets/139719193/eb45d5dd-f5a0-4153-b727-89db1a9c3a7f)
This is exactly the error

**Comment by kaerns** [comment]
Also experiencing the issue as described.

Duplicates: #980 #1036

**Comment by RU-B0** [comment]
Just wanted to comment how I fixed this issue: I found out that a mod called fusion (https://modrinth.com/mod/fusion-connected-textures) launches with OptiFine just fine. Using Midnighttigger's Default Connected Textures (https://modrinth.com/resourcepack/mt-ct-d/version/tfKvt8nl) and putting it as lowest priority on the resource pack list (but not below MC itself or fabric), it gets rid of this issue. Plus, OptiFine's implementation still works for other resource packs. Its a win - win. 

Tested with Fusion 1.0.5a and Optifine HD U I5, Optifabric 1.13.25, and FAPI 0.83.1 (+10000000000 other mods but you get the point)
